organization/_posts/2017-11-02-startup.md

```diff
@@ -0,0 +1,150 @@
+
+- #### Submission of WG papers to arXiv
+  - To be discussed at next week’s Ed Board.
+
+
+## Activity updates
+
+### Licensing
+  - Graeme heard back from Jorgen with a few questions about licenses for HepMC - it sounded like LGPL would be possible (TBC, of course!).
+  - Will have another discussion with LHCb tomorrow morning.
+  
+### Training
+  - Graeme was helping with LHCb StarterKit training this week, but due to lack of time discussion is postponed to next week.
+
+### Packaging
+  - Ben Morgan agreed to be the co-convenor of this group,
+  - Group had a [meeting this week (#13)](https://indico.cern.ch/event/674780/).
+    - We had updates on Spack and started discussing containers as used by SuperNEMO.
+    - Started to define a stack that will be used for real tests of the tools we are working with (Spack and Portage).
+    - Deployment targets:
+      - CVMFS
+      - Containers (Docker + Singularity)
+  - [Next meeting 15th November](https://indico.cern.ch/event/678307/). 
+  
+    
+## AOB
+
+### HSF/WLCG Workshop
+- Graeme, Benedikt, Michael, Ian B and Ian C will meet next Tuesday to come up with a draft agenda.
+
+### Pre-CHEP Workshop
+- Graeme sent an email to the forum list to solicit ideas.
+
+#### Proposal: PyHEP
+- Eduardo has had in mind the organisation of a workshop "PyHEP - Python in HEP" for a while. Seems such a workshop fits well with the scope of a pre-CHEP event.
+  - Proposal to organise PyHEP with HSF support. Initiative could start a series of PyHEP events.
+  - Why such a PyHEP workshop?
+    - There are certainly HEP confernces and workshops discussing computing and software.
+    - But none really devoted to the programming language increasingly chosen in analysis.
+    - Bring together users and developers.
+  - Scope
+    - Step back and review evolution of Python in the HEP community at large.
+    - Discuss - important: with a wide audience, not just LHC! - where we are going,
+             want to go, decisions to make, etc.
+     - Review the landscape of Python tools available, what we want and/or lack.
+  - Team
+    - So far interest from Jeff Templon (with whom the idea was discussed in the past).
+  - Support
+    - HSF support would make a lot of sense.
+  - Keynote presentation examples
+    - Historically-oriented review of Python in HEP “from day 1 around Nikhef”.
+   - Agenda add-ons
+     - Any tutorial, hackathon, ... ?
+
+- Proposal was very well received but need to identify the potential audience and ensure that the right people are involved in the organisation.
+  - When this is more precise, come with a strawman proposal.
+
+#### Other workshop ideas
+  - Idea of workload management was raised again, but this would have to be a follow on from the March workshop.
```

Comment:
I'd say more explicitely that this will probably be one of the topic of the forthcoming March workshop, due to the presence of both SW experts and distributed computing experts.